sunpy/physics/differential_rotation.py

```diff
@@ -173,7 +175,8 @@ def rot_hpc(x, y, tstart, tend, frame_time='synodic', rot_type='howard', **kwarg
                                          y.to(u.arcsec).value,
```

Comment:
The `_calc_P_B0_SD` call is potentially at least *partially* redundant by the frame attributes attached to the input coordinate object.